cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -592,35 +648,14 @@ public InstancesInfo getApplicationInstances(String applicationName) {
     public InstancesInfo getApplicationInstances(CloudApplication application) {
         if (application.getState()
             .equals(CloudApplication.State.STARTED)) {
-            return doGetApplicationInstances(application.getMetadata()
-                .getGuid());
+            return findApplicationInstances(getGuid(application));
         }
         return null;
     }
 
     @Override
     public List<CloudApplication> getApplications() {
-        return getApplications(true);
-    }
-
-    @Override
-    public List<CloudApplication> getApplications(boolean fetchAdditionalInfo) {
-        Map<String, Object> urlVars = new HashMap<>();
-        String urlPath = "/v2";
-        if (target != null) {
-            urlVars.put("space", target.getMetadata()
-                .getGuid());
-            urlPath = urlPath + "/spaces/{space}";
-        }
-        urlPath = urlPath + "/apps";
-        urlPath = fetchAdditionalInfo ? urlPath + "?inline-relations-depth=1" : urlPath;
-        List<Map<String, Object>> resourceList = getAllResources(urlPath, urlVars);
-        List<CloudApplication> applications = new ArrayList<>();
-        for (Map<String, Object> resource : resourceList) {
-            resource = fetchAdditionalInfo ? processApplicationResource(resource) : resource;
-            applications.add(mapCloudApplication(resource));
-        }
-        return applications;
+        return fetchListWithAuxiliaryContent(this::getApplicationResources, this::zipWithAuxiliaryApplicationContent);
```

Comment:
Even though we discussed it yesterday, I googled it today: "Translate auxiliary" :D

Reply:
I chose it for consistency, because they're also using it as a term in the v3 client. "Additional" is probably better, though, so I'll ask someone for a third opinion. Consistency vs readability?